index.bs

```diff
@@ -3194,7 +3194,7 @@ associated with or [=scoped=] to, respectively.
 
         When [=clients=], [=client platforms=], or [=authenticators=] display a {{PublicKeyCredentialEntity/name}}'s value, they should always use UI elements to provide a clear boundary around the displayed value, and not allow overflow into other elements [[css-overflow-3]].
 
-        Authenticators MAY truncate a {{PublicKeyCredentialEntity/name}} member's value so that it fits within 64 bytes, if the authenticator stores the value. See [[#sctn-strings-truncation]] about truncation and other considerations.
+        Authenticators MAY truncate a {{PublicKeyCredentialEntity/name}} member's value so that its UTF-8 encoding fits within 64 bytes, if the authenticator stores the value. See [[#sctn-strings-truncation]] about truncation and other considerations.
```

Comment:
The "UTF-8 encoding" reads oddly to me. Also, we want to be clear that we mean characters/code points here.

```suggestion
Authenticators MAY truncate characters from the end of a {{PublicKeyCredentialEntity/name}} member's value so that when the name is encoded in UTF-8, its length fits within 64 bytes, if the authenticator stores the value. See [[#sctn-strings-truncation]] about truncation and other considerations.
```

Also, is "if the authenticator stores the value" necessary?

Reply:
Thanks! I agree that "if the authenticator stores the value" wasn't necessary. I also saw some opportunities to make this language both shorter and more strict, please let me know what you think.